app/spaces/create/components/constant.tsx

```
@@ -0,0 +1,69 @@
+            color = "#77FAFF"
+            break;
+        }
+        case 'developer': {
+            icon = <CodeIcon />
+            color = "#7388FF"
+            break;
+        }
+        case 'permahub': {
+            icon = <AirTrafficControlIcon />
+            color = "#D364FF"
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)